fdbclient/RestoreWorkerInterface.actor.h

```diff
@@ -342,6 +342,31 @@ struct RestoreLoadFileRequest : TimedRequest {
 	}
 };
 
+struct RestoreSendMutationsToAppliersRequest : TimedRequest {
+	constexpr static FileIdentifier file_identifier = 68827305;
+
+	std::map<Key, UID> rangeToApplier;
```

Comment:
I noticed the type is changed from a `Standalone` to `Key` here, which might not be what you wanted.

Reply:
Never mind. They are the same.